ExportCSV.ascx.cs

```diff
@@ -154,16 +155,40 @@ static string CleanName(string name)
             return badChars.Aggregate(strName, (current, badChar) => current.Replace(badChar.ToString(), ""));
         }
 
-        void ExportData(string folder, string strFileName, string delimiter)
+        void ExportData(string folder, string strFileName, string delimiter, string initialDate, string finalDate)
         {
-            var ds = new UserDefinedTableController(_moduleId, TabId, UserInfo).GetDataSet(true);
+            DataSet ds;
+
+            if(initialDate.Length>0 && finalDate.Length > 0)
```

Comment:
This probably needs to be changed, but just as a note for performance for code in general, this might be better done using the built-in `string.IsNullOrEmpty()` method, because these are string values, and length could return a value greater than zero if there is one or more space values.

Reply:
Yes you are right. Sorry im not so experienced about how github works, how i can make this change to improve this PR.

Reply:
No worries...  In the same branch you used to create the PR, just make the update and then push again as a new commit. This will update the PR for you.  :)  Nothing more is needed.